lib/active_model_serializers/json_api_objects/json_api.rb

```diff
@@ -0,0 +1,26 @@
+module ActiveModelSerializers::JsonApiObjects
+  module JsonApi
```

Comment:
yeah, a ref would be good. 
And also mentioning that it " just adds the jsonapi object if it's configured to add it" is sufficient as long as the ref is included as well. 

The goal here is to reduce confusion for those looking at the code *for the first time*. 
They may not have the json api spec memorized, and might think, "Why is this here?"